recipe/patch_yaml/libcusolver.yaml

```diff
@@ -1,135 +1,8 @@
 ---
 if:
   name: libcusolver
```

Comment:
Note that [only `libcusolver` uses `libnvjitlink`]( https://github.com/conda-forge/libcusolver-feedstock/blob/a210ce23b096eecc30317879ee3e3d98ccb8e1dd/recipe/meta.yaml#L57 ). `libcusolver-dev` does not. So we don't need to patch the latter